INTITA/Modules/ProfileSettings/SettingsProfileViewController.swift

```diff
@@ -73,46 +87,39 @@ extension SettingsProfileViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         
-        var height: CGFloat = 0
-        
-        if indexPath.row == 0 {
-            height = 188
-        } else {
-            height = 45
-        }
-        
-        return height
+        return 45
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        if indexPath.row == 0 {
-            let headerCell = tableView.dequeueReusableCell(withIdentifier: "HeaderSettingsTableViewCell", for: indexPath) as! HeaderSettingsTableViewCell
-            
-            headerCell.delegate = self
-            return headerCell
+        let cell = tableView.dequeueReusableCell(withIdentifier: "InfoSettingProfileTableViewCell", for: indexPath) as! InfoSettingProfileTableViewCell
```

Comment:
It's not recommended to user force unwraps

Reply:
Sorry. My bad. I fixed it.